daemon/mgr/network.go

```diff
@@ -727,7 +727,5 @@ func joinOptions(endpoint *types.Endpoint) ([]libnetwork.EndpointOption, error)
 	var joinOptions []libnetwork.EndpointOption
 	// TODO: parse endpoint's links
 
-	// set priority option
-	joinOptions = append(joinOptions, libnetwork.JoinOptionPriority(nil, endpoint.Priority))
```

Comment:
we will use this feature, can you fix this bug in our libnetwork(https://github.com/alibaba/libnetwork) ?